.travis.yml

```diff
@@ -1,13 +1,13 @@
 language: python
 required: sudo
 python:
-    "2.7"
+    - "2.7"
+    - "3.2"
 virtualenv:
     system_site_packages: true
 install:
-  - sudo apt-get install python-opencv
+  - sudo -H python -m pip install opencv-python pillow
```

Comment:
That sounds fine to me, at least for now. I develop on Python 3 and OpenCV 3, and I can confirm that the tests pass on that. I still have to test with the changes to `goncalopp/master` though.